CMakeLists.txt

```diff
@@ -23,13 +25,16 @@ include_directories(${SIMPLE_WEB_SERVER_HDR})
 
 
 file(GLOB_RECURSE EXPRESS_SRC "src/*.cpp")
-add_library(express-cpp SHARED ${EXPRESS_SRC})
```

Comment:
Why not static by default, it's actually the praised way by CMake and massive projects like hunter. It in fact makes application deployment way easier I think.

But I add it per default ON in your project, even though I think it should be OFF per default.